core-common/src/main/java/eu/arrowhead/common/dto/internal/KeyValueDTO.java

```diff
@@ -0,0 +1,32 @@
+package eu.arrowhead.common.dto.internal;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+public class KeyValueDTO implements Serializable {
```

Comment:
Because it contains multiple key-value pairs, you should rename it to KeyValuesDTO, or something similar.